eventdiscovery-sdk/src/main/res/drawable/schedjoules_ic_arrow_drop_down_light.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="@dimen/schedjoules_toolbar_title_font_size"
+        android:height="@dimen/schedjoules_toolbar_title_font_size"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+            android:fillColor="@color/schedjoules_dark_text_primary"
```

Comment:
> Regarding the one drawable with an attribute color: That's not possible. Drawables can only use color resources (at least on SDK level <21). They don't support "colors by reference". That's also why we can't create drawables with "accent color" in XML. It's said that this is supported since SDK Level 21, but we still need to support older SDK levels, so there is no benefit if we use this technique.

I just tried this with `android:fillColor="?attr/colorAccent"` and it worked for pre-Lollipop emulators as well.